dandi/metadata.py

```diff
@@ -776,6 +776,7 @@ def process_ndtypes(asset, nd_types):
     technique = set()
     variables = set()
     for val in nd_types:
+        val = val.split()[0]
```

Comment:
might be weekend/monday and the cold, but I still find it difficult to identify from the long (and seemingly not specific to the issue) added test why such splitting solves some problem.  @satra could you please digest it for me a bit?